typescript/syntax/typescript.sdf3

```diff
@@ -16,4 +17,5 @@ context-free start-symbols
 
 context-free syntax
  
+  Program.Module  = Module
```

Comment:
We need to have `.Module` here, or your whole analysis is broken. I do not know why :stuck_out_tongue_closed_eyes: